modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/index/impl/AbstractTestIndexStorage.java

```diff
@@ -55,7 +56,7 @@ public int pendingCursors() {
 
     @Override
     public Cursor<RowId> get(BinaryTuple key) {
-        checkStorageClosedOrInProcessOfRebalance();
+        checkStorageClosedOrInProcessOfRebalance(true);
```

Comment:
Shall we create a wrapper over index storages (e.g. `RebalanceAwareIndexStorage`), that will perform all these checks? It's a bit frustrating that every storage implementation has to check this

Reply:
I like the idea, but right now I'm a bit time-constrained. Will it be ok with you if I create a Jira issue and we solve it later?

Reply:
yes

Reply:
I filed a Jira issue: https://issues.apache.org/jira/browse/IGNITE-21825